chargeOclock/Source/Models/Client.swift

```diff
@@ -51,4 +51,9 @@ extension Client {
 		}
 	}
 
+	func remove(completion: @escaping (Error?) -> Void) {
```

Comment:
please move this method into the class itself...
no other reason than that this method actually operates on the model object, while for example, the `all` method handles a collection of the model objects.